shared/panels/program-sidepanel/program-sidepanel.tsx

```diff
@@ -59,6 +68,32 @@ export function ProgramSidepanel() {
       />
 
       <SidePanelBody>{renderContent()}</SidePanelBody>
+      {onEditClick || !!data ? (
```

Comment:
If there is an onEdit callback but no data we can still see the footer

Reply:
good one